reflekt-plugin/src/main/kotlin/io/reflekt/plugin/analysis/processor/uses/BaseUsesProcessor.kt

```diff
@@ -4,26 +4,43 @@ import io.reflekt.plugin.analysis.models.ClassOrObjectInvokes
 import io.reflekt.plugin.analysis.models.ClassOrObjectUses
 import io.reflekt.plugin.analysis.models.SupertypesToAnnotations
 import io.reflekt.plugin.analysis.processor.Processor
+import io.reflekt.plugin.analysis.processor.fullName
 import io.reflekt.plugin.analysis.psi.annotation.getAnnotations
 import io.reflekt.plugin.analysis.psi.isSubtypeOf
-import org.jetbrains.kotlin.psi.KtClassOrObject
-import org.jetbrains.kotlin.psi.KtElement
+import org.jetbrains.kotlin.psi.*
 import org.jetbrains.kotlin.resolve.BindingContext
 
-abstract class BaseUsesProcessor<Output : Any>(override val binding: BindingContext): Processor<Output>(binding) {
-    abstract val uses: Output
+//TODO: check code generation case when invokes are not empty but uses are empty and add tests for code generation
+abstract class BaseUsesProcessor<Output : Any>(override val binding: BindingContext) : Processor<Output>(binding) {
+    // Store uses by file
+    abstract val fileToUses: HashMap<String, Output>
 
-    protected fun processClassOrObjectUses(element: KtElement, invokes: ClassOrObjectInvokes, uses: ClassOrObjectUses): ClassOrObjectUses {
+    protected fun processClassOrObjectUses(
+        element: KtElement, file: KtFile,
```

Comment:
shouldn't they be on different lines?

Reply:
oh thank you, I will create an issues about adding ktlint in this repo